### ISSUE TITLE
Type '-1' is not assignable to type 'NoteEnum' error during compilation

### ISSUE THREAD

**Issue by fablau** [opensheetmusicdisplay/opensheetmusicdisplay#1356]
Hello.

I am trying to compile the source code inside VSCode to possibly apply my own modifications and issue some pull requests to the repo, but when I compile the code, I get the following error:

```
src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts:626:126 - error TS2322: Type '-1' is not assignable to type 'NoteEnum'.

626         const noteIndices: NoteEnum[] = [NoteEnum.C, NoteEnum.D, NoteEnum.E, NoteEnum.F, NoteEnum.G, NoteEnum.A, NoteEnum.B, -1];
                                                                                                                                 ~~
Found 1 error in src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts:626
```

The exact line that's referring to is this:

https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/blob/8efb23a4887f58d989920cc2a3c6efbc506875a8/src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts#L626

And seems actually a legitimate error since the NoteEnum is so defined inside Pitch.ts:

https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/blob/8efb23a4887f58d989920cc2a3c6efbc506875a8/src/Common/DataObjects/Pitch.ts#L2

Am I missing anything?

**Comment by sschmidTU** [comment]
yes, it's on develop.
<img width="415" alt="image" src="https://user-images.githubusercontent.com/33069673/228326928-e72f5bf2-4805-4a4d-9b76-fffe1b3def00.png">

**Comment by fablau** [comment]
Yes, you are right, I see it online, but for some reason my version of GitHub Desktop doesn't fetch the last commit:

![image](https://user-images.githubusercontent.com/10005316/228328188-f3d6360d-4a94-4c9a-b92d-6c7e4060a9e6.png)

I'll figure that out.

Anyway, I removed the -1, and now it builds perfectly.

Thank you!

**Comment by sschmidTU** [comment]
@fablau That's strange. Try restarting github desktop? or just do `git pull` in the console manually.
I did force-push a commit to change the commit message, though it was before you pulled anything. Maybe Github Desktop can't deal with that, which would be very sad. Maybe try re-cloning the repository or something.

**Comment by sschmidTU** [comment]
Great, thanks for the contributions and reports, very much appreciated!

**Comment by fablau** [comment]
Of course! I plan to work on this project pretty actively in the coming months ;)
